src/vm/codeman.h

```diff
@@ -140,6 +140,10 @@ typedef struct _hpCodeHdr
     PTR_EE_ILEXCEPTION  phdrJitEHInfo;
     PTR_BYTE            phdrJitGCInfo;
 
+#if defined(FEATURE_GDBJIT)
+    VOID*            pCalledMethods;
```

Comment:
We considered this way but it requires implementing disassembler for each supported platform. What we need is only a list of native call target thunks to generate symbolic information that makes lldb happy. 
As far as I understand, precompiled specific code is wrapped by FEATURE_PREJIT, so we could just wrap our additions by negation of this flag.